plugins/key/admin.key.js

```diff
@@ -35,38 +33,32 @@ const setupAdminKey = async function (fastify) {
     const adminPwd = generatePassword();
     const hashPwd = await createHash(adminPwd);
 
-    //Create a user object to use to create the user in the database and access token
+    //Create a default admin account
     const userObj = {
       uuid: uuid,
-      name: "Admin",
-      email: "admin@localhost",
+      name: 'Admin',
+      email: 'admin@localhost',
     };
 
     const registerStmt = fastify.db.prepare(
-      "INSERT INTO users (uuid, name, email, password, active, jwt_id, created_at, updated_at) VALUES (?, ?, ?, ?, ?, ?, strftime('%Y-%m-%dT%H:%M:%fZ','now'), strftime('%Y-%m-%dT%H:%M:%fZ','now')) RETURNING uuid, name, email, active, created_at, updated_at;"
-    );
-    const user = registerStmt.get(
-      uuid,
-      userObj.name,
-      userObj.email,
-      hashPwd,
-      1,
-      ""
+      "INSERT INTO admin (uuid, name, email, password, active, jwt_id, created_at, updated_at) VALUES (?, ?, ?, ?, ?, ?, strftime('%Y-%m-%dT%H:%M:%fZ','now'), strftime('%Y-%m-%dT%H:%M:%fZ','now')) RETURNING uuid, name, email, active, created_at, updated_at;"
     );
+    const user = registerStmt.get(uuid, userObj.name, userObj.email, hashPwd, 1, '');
 
     //export admin password to a file. Admin password is only exported once on server startup and can be traded for an access token
     writeFileSync(config.ADMINKEYPATH, `admin password: ${adminPwd}`);
-    fastify.log.info(`Generating Admin API Key: ${config.KEYPATH}...`);
+    fastify.log.info(`Generating Admin API Key: ${config.ADMINKEYPATH}...`);
+    fastify.log.info(`Admin Password is: ${adminPwd}`);
```

Comment:
This is helpful, I'll likely update the doc to reflect that you can get the password here in log a well.